posthog/test/base.py

```diff
@@ -1000,7 +1000,7 @@ def wrapped(self, *args, **kwargs):
             fn(self, *args, **kwargs)
 
         for query in queries:
-            if "FROM system.columns" not in query:
+            if "FROM system.columns" not in query and 'FROM "posthog_instancesetting"' not in query:
```

Comment:
these should never be in clickhouse, so _I think_ shouldn't be necessary in all these files here? 

You just need it here -> https://github.com/PostHog/posthog/blob/master/posthog/test/base.py#L677 (unless of course I'm mistaken how this is setup)

I added a custom query matcher recently, and you could potentially build on top of that.

But yes love this, this is eggcelent!

Reply:
Ah, yes, this was a lazy, dump it everywhere to open the PR change 🤣